src/components/utils/test/message_queue_test.cc

```diff
@@ -209,7 +209,7 @@ TEST_F(MessageQueueTest,
   threads::Thread* thread1 =
       threads::CreateThread("test thread", new ShutDownQueueDelegate(job_sync));
   thread1->start();
-  job_sync.WaitJobDone(100);
+  job_sync.WaitJobDone(2000);
```

Comment:
@orhan-mehmedov Why yo increased  this time. In my opinion UT should be extremely fast. 2 seconds for one unsuccessful case it  is too long